tests/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM dockerswarm/dind:1.8.1
+MAINTAINER Joffrey F <joffrey@docker.com>
+
+RUN mkdir /home/docker-py
+WORKDIR /home/docker-py
+
+RUN apt-get update
+RUN apt-get install -y python python-setuptools && easy_install pip
```

Comment:
Couple minor suggestions here:

* [best practices](https://docs.docker.com/articles/dockerfile_best-practices/#run) says to make these a single line, otherwise you can run into a situation where the update is cached without the available package you need to install
* instead of `easy_install pip` why not add `python-pip`  to the apt-get install line? If you want a newer version https://pip.pypa.io/en/latest/installing.html#install-pip suggests their install script.